packages/transformers/babel/src/getBabelTargets.js

```diff
@@ -66,6 +66,7 @@ export default async function getBabelTargets(
       browsers != null &&
       !Array.isArray(browsers)
     ) {
+      // flowlint sketchy-null-string: off
       let env = process.env.NODE_ENV || 'development';
```

Comment:
@padmaia @devongovett looks like this is using the real environment's `NODE_ENV`. 

@thebriando `process.env.NODE_ENV ?? 'development` addresses this, though it changes the semantics, probably for the better. In the future, we can avoid disabling lint by making the falsy check explicit like `(foo !== '' && foo != null) ? foo : 'fallback'`